app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -69,4 +71,11 @@ class BindingTariffClassificationConnector @Inject()(configuration: AppConfig, c
     client.GET[Seq[Event]](url)
   }
 
+  def search(search: Search, sort: Sort)(implicit hc: HeaderCarrier, binder: QueryStringBindable[String]): Future[Seq[Case]] = {
+    val searchString = if(!search.isEmpty) "&" + Search.bindable.unbind("", search) else ""
```

Comment:
looking at how you created the `Search` object, I would replace `! ... isEmpty` with `isDefined` (more readable I think)

Reply:
Done